share/instruments/instruments.xml

```diff
@@ -9987,15 +9962,15 @@
                   <musicXMLid>pluck.guitar</musicXMLid>
                   <StringData>
                         <frets>19</frets>
-                        <string open="1">31</string>
-                        <string open="1">32</string>
-                        <string open="1">33</string>
-                        <string open="1">34</string>
-                        <string open="1">35</string>
-                        <string open="1">36</string>
-                        <string open="1">37</string>
-                        <string open="1">38</string>
-                        <string open="1">39</string>
+                        <string>31</string>
+                        <string>32</string>
+                        <string>33</string>
+                        <string>34</string>
+                        <string>35</string>
+                        <string>36</string>
+                        <string>37</string>
+                        <string>38</string>
+                        <string>39</string>
```

Comment:
This looks fishy, double check needed, but I don't think there's any guitar with 10 fretted strings in a row that are just one semiitone appart?